test/gurka/test_instructions_obvious_maneuver.cc

```diff
@@ -135,13 +135,13 @@ TEST_F(InstructionsObviousManeuver, NotObviousBecauseSameNameIntersectingEdgeRam
   // Verify start maneuver prior to the continune maneuver
   gurka::assert::raw::expect_instructions_at_maneuver_index(result, maneuver_index,
                                                             "Drive east on US 322.", "",
-                                                            "Drive east on US 3 22.",
+                                                            "Drive east on US 322.",
```

Comment:
Are these tests updated because they're using netherlands admin?

Reply:
@danpaz yes, exactly

Reply:
🤔 I'm wondering if this formatting should really be based on the request language and not the country. In Canada and even in Europe I think it's desired to pronounce the digits separately, e.g. the example in one of the tests below `A120` seems like it'd be a regression if the instruction language is in English rather than French. @st4141 as you wrote the original issue https://github.com/valhalla/valhalla/issues/2612 do you have any thoughts on this?

Reply:
@danpaz yes, I was thinking the same thing about the language in the future. However, right now I just wanted to fix the current issue.

Reply:
@danpaz I guess the formatting should be based on the language of the request as you said, not on the country where the road is.

Reply:
@danpaz also, tested the `A120` phrase and it sounded appropriate.

Reply:
@danpaz @st4141 Currently, the formatter is based on location. I will submit an issue to enhance to use language. However, this PR fixes the number formatting for outside the US since that should have only applied to the US and is not English specific

Reply:
> @danpaz @st4141 Currently, the formatter is based on location. I will submit an issue to enhance to use language. However, this PR fixes the number formatting for outside the US since that should have only applied to the US.

👍 sounds good to tackle this in a separate issue.

Reply:
Is there an issue for using request language instead of country for the text formatter? Rest lgtm.